app/views/layouts/_navbar.html.erb

```diff
@@ -1,31 +1,39 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark " data-bs-theme="dark">
+<nav class="navbar navbar-expand-lg navbar-light", style="background-color: #d9d9d9;">
   <div class="container-fluid">
-    <a class="navbar-brand" href="#">
+    <a class="navbar-brand  me-2" href="/">
       <%= image_tag 'logo.png', alt: 'CHAMPS', width: 60 %>
     </a>
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
+    <button
+      class="navbar-toggler"
+      type="button"
+      data-mdb-toggle="collapse"
+      data-mdb-target="#navbarButtonsExample"
+      aria-controls="navbarButtonsExample"
+      aria-expanded="false"
+      aria-label="Toggle navigation"
+    >
```

Comment:
This is not looking good
move `class="navbar-toggler"` after buttton.
then fix alignment and move `>` after area-label.